### ISSUE TITLE
BUG: no SMTP-Auth is being used when using credentials from IMAP

### ISSUE THREAD

**Issue by computersalat** [Intermesh/groupoffice#1066]
Using _groupoffice-6.7.50_
When I add an email account, there is an option for SMTP Server to use `IMAP-Anmeldeinformationen verwenden`.
When this option is used, then no smtp auth is being used when sending mail.
smtp auth is only used, when I do _NOT_ use `IMAP-Anmeldeinformationen verwenden` and define `Authentifizierung benutzen`

So IMHO this feature is **_NOT_** working as expected and hence it is a BUG.

**Comment by computersalat** [comment]
Error message when using credentials from IMAP to send mail:
```
>> DATA

<< 250 2.1.0 Ok

<< 554 5.7.1 <xxxxxx@gmail.com>: Relay access denied

!! Expected response code 250/251/252 but got code "554", with message "554 5.7.1 <xxxxxx@gmail.com>: Relay access denied
" (code: 554)
<< 554 5.5.1 Error: no valid recipients

!! Expected response code 354 but got code "554", with message "554 5.5.1 Error: no valid recipients
" (code: 554)
Log data:
++ Starting GO\Email\Transport
<< 220 mx.domain.email ESMTP

>> EHLO go.domain.services

<< 250-mx.domain.email
250-PIPELINING
250-SIZE
250-ETRN
250-AUTH PLAIN LOGIN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250-DSN
250-SMTPUTF8
250 CHUNKING

++ GO\Email\Transport started
>> MAIL FROM:<info@yyyyyy.net>

>> RCPT TO:<xxxxxx@gmail.com>

>> DATA

<< 250 2.1.0 Ok

<< 554 5.7.1 <xxxxxx@gmail.com>: Relay access denied
!! Expected response code 250/251/252 but got code "554", with message "554 5.7.1 <xxxxxx@gmail.com>: Relay access denied
" (code: 554)
<< 554 5.5.1 Error: no valid recipients

!! Expected response code 354 but got code "554", with message "554 5.5.1 Error: no valid recipients
" (code: 554)
!! Expected response code 354 but got code "554", with message "554 5.5.1 Error: no valid recipients
"
Log data:
++ Starting GO\Email\Transport
<< 220 mx.domain.email ESMTP

>> EHLO go.domain.services

<< 250-mx.domain.email
250-PIPELINING
250-SIZE
250-ETRN
250-AUTH PLAIN LOGIN
250-ENHANCEDSTATUSCODES
250-8BITMIME
250-DSN
250-SMTPUTF8
250 CHUNKING

++ GO\Email\Transport started
>> MAIL FROM:<info@yyyyyy.net>

>> RCPT TO:<xxxxxx@gmail.com>

>> DATA

<< 250 2.1.0 Ok

<< 554 5.7.1 <xxxxxx@gmail.com>: Relay access denied
!! Expected response code 250/251/252 but got code "554", with message "554 5.7.1 <xxxxxx@gmail.com>: Relay access denied
" (code: 554)
<< 554 5.5.1 Error: no valid recipients

!! Expected response code 354 but got code "554", with message "554 5.5.1 Error: no valid recipients
" (code: 554) (code: 554)
```
so no smtp_auth is being used

**Comment by computersalat** [comment]
> This is indeed buggy in 6.7 but has been fixed in 6.8. Can you upgrade?

@mschering 
I am on github rev b66c96863 (upstream/6.7.x) branch. When I switch branch to e6eedbbc4 (tag: v6.8.19) and go to upgrade I run into the following Error:
```
Error: ErrorException in /srv/www/vhosts/go.xxxxxx.com/groupoffice/www/go/core/fs/File.php at line 238:
file_get_contents(/srv/www/vhosts/go.xxxxxx.com/groupoffice/www/go/modules/business/license/model/License.php):
Failed to open stream: Datei oder Verzeichnis nicht gefunden
```
where do I get **_modules/business_** into my git checkout ?
Or do I need to do a fresh install from scratch with 6.8.x ?

**Comment by mschering** [comment]
Please install using the tarball, debian package or docker.

**Comment by computersalat** [comment]
> Please install using the tarball, debian package or docker.

@mschering 
Yes, I can do, but the devel env with git source did work nicely with 6.7.x branch. Why isn't it with 6.8.x ?
What am I missing ?

**Comment by mschering** [comment]
6.8 should work too. But you don't have access to the business repo. so you would have to copy that in from the tarball. I guess it was missing when you made the switch.

**Comment by mschering** [comment]
It is required if you use the pro version with modules. If not then it should not be required. Try enable debugging to get a stack trace so you can see where it is included.

**Comment by computersalat** [comment]
If I do a fresh install on tag v6.8.19 I get the same error. That step where you can skip _license install_ is not showing up.
```
Create an administrator account

file_get_contents(/srv/www/vhosts/go.xxxxxx.com/groupoffice/www/go/modules/business/license/model/License.php): Failed to open stream: Datei oder Verzeichnis nicht gefunden
```

Is it possible that the installer vor 6.8 thinks it is a pro version install because ioncube is installed on that system ?

**Comment by computersalat** [comment]
> This is indeed buggy in 6.7 but has been fixed in 6.8. Can you upgrade?

I tested with fresh 6.8.19 tarball installation.
works now as expected (IMAP credentials do smtp_auth)
